Slim/Formats/APE.pm

```diff
@@ -36,10 +36,13 @@ use Audio::Scan;
 
 my %tagMapping = (
 	'TRACK'	       => 'TRACKNUM',
+	'WORK'         => 'WORK',
+	'GROUPING'     => 'GROUPING',
```

Comment:
I thought that too, but since there were already identical mappings for other tags, I assumed I was missing something. But since you feel the same, I'll remove.